protos/vehicle_data.proto

```diff
@@ -231,6 +424,28 @@ message Value {
     ChargingState charging_value = 8;
     ShiftState shift_state_value = 9;
     bool invalid = 10;
+    LaneAssistLevelEnum lane_assist_level_value = 11;
+    ScheduledChargingModeEnum scheduled_charging_mode_value = 12;
+    SentryModeStateEnum sentry_mode_state_value = 13;
+    SpeedAssistLevelEnum speed_assist_level_value = 14;
+    BMSStateEnum bms_state_value = 15;
+    BuckleStatusEnum buckle_status_value = 16;
+    CarTypeEnum car_type_value = 17;
+    ChargePortEnum charge_port_value = 18;
+    ChargePortLatchEnum charge_port_latch_value = 19;
+    CruiseStateEnum cruise_state_value = 20;
+    Doors door_value = 21;
+    DriveInverterStateEnum drive_inverter_state_value = 22;
+    HvilStatusEnum hvil_status_value = 23;
+    WindowStateEnum window_state_value = 24;
+    SeatFoldPositionEnum seat_fold_position_value = 25;
+    TractorAirStatusEnum tractor_air_status_value = 26;
+    FollowDistanceEnum follow_distance_value = 27;
+    ForwardCollisionSensitivityEnum forward_collision_sensitivity_value = 28;
+    GuestModeMobileAccessStateEnum guest_mode_mobile_access_state_value = 29;
+    TrailerAirStatusEnum trailer_air_status_value = 30;
+    Time time_value = 31;
```

Comment:
why can't we use protobuf timestamp here?

Reply:
Time doesn't have a date associated with it unfortunately.

Reply:
nit: Perhaps a comment in the protobuf message saying somethig like `Time represents current day's time in 24 hour format` to make the usage clearer?